tsdb/chunkenc/chunk_test.go

```diff
@@ -217,3 +217,46 @@ func benchmarkAppender(b *testing.B, newChunk func() Chunk) {
 
 	fmt.Println("num", b.N, "created chunks", len(chunks))
 }
+
+func TestXor(t *testing.T) {
+	timestamp := int64(1234123324)
+	v := 83665193758603.123
+	num := 10
+	var samples []pair
+	for i := 0; i < num; i++ {
+		timestamp += int64(1000)
+		v += float64(100)
+		samples = append(samples, pair{t: timestamp, v: v})
+	}
+
+	chunk := NewXORChunk()
+	a, err := chunk.Appender()
+	if err != nil {
+		t.Fatalf("get appender: %s", err)
+	}
+	for _, p := range samples {
+		a.Append(p.t, p.v)
+	}
+
+	res := make([]pair, 0)
```

Comment:
```suggestion
	res := make([]pair, num)
```

Reply:
If I write this way, I need to introduce an additional 'i' index into the code, which will make it more troublesome. I
n addition, "num = 5" memory consumption can be ignored, 5 is enough to verify the accuracy of the algorithm

Reply:
```suggestion
	res := make([]pair, num, 0)
```
then :)

Reply:
Anyway that does not make a difference, you can ignore this

Reply:
![image](https://user-images.githubusercontent.com/9583245/75447841-18df8880-59a5-11ea-90c2-15136b9ec29e.png)
done,Thanks